src/components/FeatureViews/Accounts/Accounts.js

```diff
@@ -292,20 +294,26 @@ const Accounts = () => {
                         <Grid item md={1} />
                         <Grid container item md={3}>
                             <Grid className="toggleView" item md={6}>
-                                <Button
-                                    className={`btn list ${viewToggle && "active"}`}
-                                    onClick={setView(true)}>
-                                    <ListView className={`icon ${viewToggle && "active"}`} />
-                                List View
-                            </Button>
+                                <ResponsiveButton
+                                    // className={`btn list ${viewToggle && "active"}`}
```

Comment:
Why are the className and startIcon commented out in these buttons on line 297 and 308?